### DISCUSSION TITLE
申请退款:着急使用，但一直未给处理

### DISCUSSION THREAD

**Discussion by Niejingchun** [getlantern/lantern-forum-cn#967] [General]
账户已经购买Pro版，但验证时邮箱一直无法收到复原码，垃圾箱查看了了没有，因此登陆不上Pro版。请解决。购买pro版的邮箱：niejingchun@yeah.net。如何不行，复原码也可发送至niejingchun@gmail.com。 请帮忙解决。

**Comment by Niejingchun** [comment]
这是支付截图。请帮忙尽快处理一下。 
![Screenshot_2024-10-17-05-46-07-612_com discoverfinancial mobile](https://github.com/user-attachments/assets/91424e49-3425-45c5-8b69-cb68722d64e3)

**Comment by Niejingchun** [comment]
麻烦给尽快处理一下，着急用。如果处理不了，麻烦给退款。

**Comment by MaxMKMax** [comment]
@Niejingchun 您好. 马上给您提交问题上去处理. 谢谢您的耐心等待.

**Comment by Niejingchun** [comment]
能解决吗？着急用

**Comment by Niejingchun** [comment]
我申请退款。请给退款。

**Reply by MaxMKMax** [reply]
@Niejingchun 不好意思啊. 今天我让负责人帮您解决. 他前几天病了, 请了病假. 所以联系不上. 非常抱歉啊.

**Comment by Niejingchun** [comment]
申请退款:niejingchun@yeah.net。着急使用，已购买其他vpn产品。请尽快安排退款

**Reply by MaxMKMax** [reply]
@Niejingchun 好的. 明白了.

**Comment by greySkiesMarsh** [comment]
Hi @Niejingchun 

I'm looking into your refund request, but I'm having trouble tracking down your transaction. Did you make your purchase through the google play store, or directly through the app? Unfortunately, the bank receipt that you provided doesn't give us much to go on.

**Reply by Niejingchun** [reply]
I made a direct purchase through the application rather than using the Google Play Store. I just upgraded it within the application, which prompted me to provide my credit card details. The credit card number used for payment is 6011007504550624, and the name associated with it is JINGCHUN NIE.  Is there any additional information you require?

**Comment by Niejingchun** [comment]
还没给退吗？

**Reply by MaxMKMax** [reply]
@Niejingchun 支付团队还退不了. 需要您提供一下Google的支付单据. 应该您的Google邮箱里会有.

**Reply by Niejingchun** [reply]
还没退为什么不给反馈一下呢？一个问题本来几分钟解决的问题，来来回回两周了都解决不了。我本来是想着着急用临时换个别的。你们这处理问题的方式怎么让人继续用？
我再解释下，第一，我注册的邮箱是niejingchun@yeah.net。我起初反应的问题就是邮箱收件问题，没有收到你们的那个注册码。我这个邮箱也没有收到过支付单据。支付完就结束了。没有完成注册。我这才联系你们解决。第二，我已经解释过了。我不是在Google play购买的，就是在APP里直接点Renew购买的。我支付的截图就是上面的那个。那上面有支付的时间和支付人，支付的卡号是Discover信用卡:6011007504550624。你还需要什么信息呢？

**Reply by Niejingchun** [reply]
到底什么时候给退？

**Comment by Niejingchun** [comment]
到底什么时候给退？

**Reply by MaxMKMax** [reply]
@Niejingchun 您好. 已经帮您提交退款申请了. 请您等待回复.

**Reply by liuh23** [reply]
我的退款呢？
反馈多少次了，一直不回复
这是我的反馈过程链接https://github.com/getlantern/lantern-forum-cn/discussions/973

**Comment by MaxMKMax** [comment]
@Niejingchun 退款的流程不是我弄的. 我只是中间人负责传达信息. 请您等等. 不好意思啊.

**Reply by Niejingchun** [reply]
什么进展？又没消息了

**Comment by Niejingchun** [comment]
怎么又没消息了？退款给处理了吗？都一个月了

**Comment by Niejingchun** [comment]
给办理退款了吗？什么时候给处理

**Reply by Raymondlee321** [reply]
死心吧，这团队就这样。

**Comment by Niejingchun** [comment]
你们装作没看见是吗？太过分了吧。什么时候退款

**Comment by Niejingchun** [comment]
给退款

**Comment by bobobun212** [comment]
退个Der阿，我到16年那，你见过那个要跑路的公司还给用户退款的

**Comment by Niejingchun** [comment]
到底什么时候给退？两个月了
